src/winetricks

```diff
@@ -11971,6 +11971,65 @@ __EOF__
 
 #----------------------------------------------------------------
 
+w_metadata office2013pro apps \
+    title="Microsoft Office 2013 Professional" \
+    publisher="Microsoft" \
+    year="2013" \
+    media="cd" \
```

Comment:
@austin987 This verb doesn't use physical disc or cached image (it *downloads* an ISO image). Would it be better to specify `media="download"` instead?

Reply:
I haven't dug too deep to see if it would cause problems (e.g. with iso parts), but I suspect you want download.

That said, I'm about 98% it should work as download rather than cd.

Reply:
a4c5879e28b5e12a199b222d4a6bd52ee3195915